oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -63,6 +63,8 @@ def __init__(self, server_hardware, server_hardware_types,
         base_resource = server_profile
         self.redfish["@odata.type"] = self.get_odata_type()
         self.redfish["Id"] = base_resource["uuid"]
+        if base_resource["description"]:
```

Comment:
Description is a string, since it is supported it should be listed even when it is empty.

Reply:
Done.